File: 1C1.py
n=int(input())
for i in range(1,n+1):
    i1,i2,i3=map(int, input().split()) 
sum1=0
sum2=0
sum3=0
for i in range(1,n+1):
    sum1+=i1
    sum2+=i2
    sum3+=i3
if(sum1==0 and sum2==0 and sum3==0):
    print("Yes")
else:
    print("No")
print(sum1)

File: 1C1.c
#include<stdio.h>
int main(){
    int n;
    scanf("%d",&n);
    int sum1=0,sum2=0,sum3=0;
    for(int i=1;i<=n;i++){
    	int i1,i2,i3;
        scanf("%d %d %d",&i1,&i2,&i3);
        sum1+=i1;
        sum2+=i2;
        sum3+=i3;
    }
    if(sum1==0 && sum2==0 && sum3==0){
        printf("Yes");
    }else{
        printf("No");
    }
    return 0;
}